indico/modules/search/client/js/components/SearchApp.jsx

```diff
@@ -330,9 +330,11 @@ export default function SearchApp({category, eventId, isAdmin}) {
           sortOptions={options?.sortOptions || []}
           onSortChange={value => handleQuery(value, 'sort')}
         />
-        {!isAnyLoading && (
-          <ResultHeader query={q} hasResults={!!results.total} categoryTitle={category?.title} />
-        )}
+        <div aria-live="polite">
```

Comment:
should the div be there all the time or only when the header is shown?

Reply:
It must be present at all times.